ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -313,6 +313,8 @@ library
                      , ouroboros-network-framework
                      , ouroboros-network
 
+                     , canonical-json
+                     , aeson
```

Comment:
Can someone comment on if adding these deps is acceptable or not?

Reply:
I just commented elsewhere that I think we can and should avoid them, in favor of `ToCBOR`/`FromCBOR`. The primary cost will be a hopefully small upstream PR to `cardano-ledger-specs`, if I have anticipated things correctly.